api/helpers/things.js

```diff
@@ -331,6 +344,58 @@ const isValidURL = string => {
   return res !== null;
 };
 
+async function createLocalizedRecord(data, thingid) {
+  var records = [];
+  for (var i = 0; i < SUPPORTED_LANGUAGES.length; i++) {
+    const language = SUPPORTED_LANGUAGES[i];
+
+    if (language.twoLetterCode !== data.language) {
+      const item = {
+        body: '',
+        title: '',
+        description: '',
+        language: language.twoLetterCode,
+        thingid: thingid
```

Comment:
@paninee we need to add timestamp data here

Reply:
done